backend/api/quivr_api/modules/brain/entity/brain_entity.py

```diff
@@ -63,6 +64,7 @@ class Brain(AsyncAttrs, SQLModel, table=True):
     prompt: Prompt | None = Relationship(  # noqa: f821
         back_populates="brain", sa_relationship_kwargs={"lazy": "joined"}
     )
+    knowledge_brain: list[KnowledgeBrain] = Relationship(back_populates="brain")
```

Comment:
That's not how many-to-many model work in sqlmodel. Take a look at : 

https://sqlmodel.tiangolo.com/tutorial/many-to-many/create-data/